stdlib/2.7/__builtin__.pyi

```diff
@@ -401,7 +403,7 @@ class bytearray(Sequence[int]):
     def translate(self, table: str) -> bytearray: ...
     def upper(self) -> bytearray: ...
     def zfill(self, width: int) -> bytearray: ...
-    @staticmethod
+    @classmethod
     def fromhex(self, x: str) -> bytearray: ...
```

Comment:
In fact bytes.fromhex() is a static method. Also, the first arg should be named cls.